composer.json

```diff
@@ -13,24 +13,24 @@
     ],
     "minimum-stability": "dev",
     "require": {
-        "symfony/framework-bundle": "~2.1",
+        "symfony/framework-bundle": "~2.2",
         "jms/translation-bundle": "~1.0"
     },
     "require-dev": {
-        "symfony/framework-bundle": "<2.1.5",
-        "symfony/css-selector": "<2.1.5",
-        "symfony/http-kernel": "<2.1.5",
-        "symfony/validator": "<2.1.5",
-        "symfony/browser-kit": "<2.1.5",
-        "symfony/finder": "<2.1.5",
-        "symfony/form": "<2.1.5",
-        "symfony/yaml": "<2.1.5",
-        "symfony/process": "<2.1.5",
-        "symfony/routing": "<2.1.5",
-        "symfony/http-foundation": "<2.1.5",
-        "symfony/class-loader": "<2.1.5",
-        "symfony/twig-bundle": "<2.1.5",
-        "sensio/framework-extra-bundle": "<2.1.5",
+        "symfony/framework-bundle": "<2.2.12",
+        "symfony/css-selector": "<2.2.12",
+        "symfony/http-kernel": "<2.2.12",
+        "symfony/validator": "<2.2.12",
```

Comment:
@schmittjoh but this means that you never test with current Symfony versions.

Having a travis job using the lowest deps would be a much more sensible approach